fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1625,6 +1625,10 @@ public String getCatalog() {
     @VariableMgr.VarAttr(name = DATACACHE_EVICT_PROBABILITY, flag = VariableMgr.INVISIBLE)
     private int datacacheEvictProbability = 100;
 
+    private int datacachePriority = 0;
+
+    private long datacacheTTLSeconds = 0L;
+
```

Comment:
Is there any better ways to pass this option instead of adding two new session variables?

Reply:
Could you provide any advice?